packages/taco/test/conditions/base/rpc.test.ts

```diff
@@ -94,5 +94,41 @@ describe('validation', () => {
         },
       });
     });
+
+    it('rejects empty parameters', () => {
```

Comment:
This is fine for now, but something to note/update as we expand to allow other rpc methods.